jumanji/environments/combinatorial/utils_test.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from jax import numpy as jnp
+
+from jumanji.environments.combinatorial.utils import get_coordinates_augmentations
+
+
+def test_utils__get_coordinates_augmentations() -> None:
+    """Checks that the augmentations of a given instance problem is computed properly."""
+    coordinates = jnp.array([[0.65, 0.85], [0.18, 0.06], [0.41, 0.19], [0.92, 0.27]])
+
+    expected_augmentations = jnp.array(
```

Comment:
Can we make coordindates and expected_augmentations fixtures? I know they are only used in this test, but IMO it will be easier to read and expand these tests